quickstart-node/src/services/mail.service.ts

```diff
@@ -0,0 +1,28 @@
+import { LoggerService } from '@services/monday-code';
+
+export class MailService {
+  // eslint-disable-next-line no-use-before-define
+  private static instance: MailService;
+
+  private constructor() {}
+
+  static getInstance(): MailService {
+    if (!this.instance) {
+      this.instance = new MailService();
+    }
+
+    return this.instance;
+  }
+
+  sendMail(mailAddress: string, message: string): Promise<boolean> {
+    return new Promise((resolve, reject) => {
+      try {
+        // Put here send email logic
+        LoggerService.getInstance().info(`Sending mail to ${mailAddress} with content: ${message}`);
+        resolve(true);
```

Comment:
If you don't really send an email, it's better to remove it. BTW, you can send a monday notification instead, using graphql - https://developer.monday.com/api-reference/reference/notification#create-a-notification

Reply:
I don't agree. This acts as a super easy example to set up that uses every service that monday code offers, even the queue service. We don't want it to be complicated and the developer can easily this if this works on the logs screen